CSS/README.md

```diff
@@ -212,9 +275,9 @@
 - [ ] - Specificity
 - [ ] - Box model
 - [ ] - Margin collapsing
-- [ ] - Box-sizing (border-box, content-box)
 - [ ] - FOUC (flash of unstyled content)
 - [ ] - CSS Methodologies (BEM, OOCSS etc,.)
+- [ ] - RWD (Responsive Web Design)
```

Comment:
+1 Q: Responsive Web Design vs Adaptive design

Reply:
Q: 25 covers that.